### PR TITLE
Add validate reference in bam

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Description

_Give your PR a **concise** yet **descriptive** title_

----

### Checklist (never delete this)

- [ ] Each method/class/object/trait should have scaladocs
- [ ] Added methods are unit tested
- [ ] Test coverage may not drop
- [ ] Documentation should be updated if required

### CODE REVIEW

File: src/main/scala/nl/biopet/utils/ngs/bam/package.scala [hunk 1]
```diff
@@ -194,4 +194,25 @@ package object bam {
     reader.close()
     dict
   }
+
+  /**
+    * This will retrieve the [[SAMSequenceDictionary]] from the bam file.
+    * When `referenceFasta is given he will validate this against the bam file.`
+    */
+  def validateReferenceInBam(
```
Comment (ffinfo) [inline src/main/scala/nl/biopet/utils/ngs/bam/package.scala]: Can this method be split up in 2 method so it also accepts `SamHeader` as input?

**Review by ffinfo — CHANGES_REQUESTED** [review]

File: src/main/scala/nl/biopet/utils/ngs/bam/package.scala (hunk 1 above)
Comment (rhpvorderman) [reply src/main/scala/nl/biopet/utils/ngs/bam/package.scala]: On it

Comment (ffinfo) [reply src/main/scala/nl/biopet/utils/ngs/bam/package.scala]: 👍

File: src/main/scala/nl/biopet/utils/ngs/bam/package.scala [hunk 2]
```diff
@@ -194,4 +189,30 @@ package object bam {
...
+      bamFile: File,
+      referenceFasta: Option[File]): SAMSequenceDictionary = {
```
Comment (ffinfo) [inline src/main/scala/nl/biopet/utils/ngs/bam/package.scala]: Maybe in the utils this should be a File instead of Option[File]?

**Review by ffinfo — CHANGES_REQUESTED** [review]
Forgot to mention, can this be tested?

**Review by ffinfo — APPROVED** [review]

**Pull request merged**